tests/transpiler/test_transpiler.py

```diff
@@ -101,7 +102,7 @@ def test_from_cirq_bad_package(bell_circuit, item):
         transpile(circuit, item)
 
 
-@pytest.mark.parametrize("program", ["Not a circuit", None])
+@pytest.mark.parametrize("program", [None])
```

Comment:
What exception is now raised if passed "Not a circuit"?

Reply:
Its the same exception, I think I removed it for some testing, will add back